_posts/en/newsletters/2020-06-10-newsletter.md

```diff
@@ -0,0 +1,308 @@
+    bitcoin transactions][topic psbt] are being used in size-constrained
+    media (e.g., QR codes, see [Newsletter #96][news96 qr]) the
+    potentially significant increase in data size may require abandoning
+    the protocol or switching to higher-capacity media.
+
+    As of this writing, we're unaware of any other hardware wallets
+    having made the same change as Trezor.  Although Trezor's approach
+    does maximize user safety against this attack, there are also
+    reasonable arguments for hardware wallets continuing to use segwit's
+    signed UTXO values rather than breaking existing software and
+    expending additional resources on legacy-style value verification:
+
+    - **Attack already well-known for three years:** Sanders publicly
+      described the attack almost three years ago and it's been
+      mentioned or alluded to by others on the [mailing list][lau
+      sighash2], in [BIPs][bip341 all amounts], and in Newsletters
+      [#11][news11 sighash] and [#46][news46 digest changes], among
+      other discussions.  Although the absence of any known exploits in
+      the past three years doesn't mean the attack isn't important, it
+      does point to a previous lack of urgency.  Perhaps it's the case
+      that any attacker who gains access to the software controller for
+      a hardware wallet is more likely to perform a different
+      attack---such as an [address substitution attack][hw
+      security]---that pays the attacker directly rather than overpaying
+      transaction fees.
+
+    - **Double signing can also lead to spending twice:** the attack
+      depends on using compromised software to get a hardware wallet
+      user to authorize two slightly different transactions (each with
+      two inputs) that look identical to the user.  Yet the same
+      compromised software could show the user two completely different
+      transactions (each spending a different input) that look
+      identical, resulting in paying the same recipient twice.  Both
+      attacks are indistinguishable from the user's perspective but the
+      fix for the fee overpayment attack doesn't fix the [spending twice
+      attack][maxwell spend twice].
+
+    - **Multisig setups may require multiple compromises:** to perform
+      the attack against funds secured by multisig with multiple
+      hardware wallets, each signer that is needed to meet the minimal
+      threshold (e.g. "2" in a "2-of-3" multisig) would need to be
+      tricked into signing the same two transaction variations.  For
+      thresholds that include an online wallet which knows the value of
+      the UTXO it's signing for (e.g. a policy-based remote signer), the
+      attack only works if that online wallet is also compromised.
+
+    A long-term solution to the attack is to change the transaction
+    digest so that each signature in a transaction commits to the value
+    of all UTXOs being spent in that transaction, making the signature
```

Comment:
It is a dense sentence. Suggest breaking up.
```suggestion
    of all UTXOs being spent in that transaction. This makes the signature
```

Reply:
With this proposed change, we'd have two sentences in a row that start with "This".  I'll make a slightly more invasive rephrasing to avoid that.